src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/Semantic/TestFiles/RazorSemanticTokensTests/Components_AreColored.txt

```diff
@@ -26,7 +26,6 @@
 113,1,HTML Tag Delimiter,
 124,1,HTML Tag Delimiter,
 125,7,RazorComponentElement,
-132,1,HTML Tag Delimiter,
```

Comment:
Just when you think you might understand classification, along comes this change, which is apparently caused by the changes in this PR, but I am definitely not smart enought to work out why!

Reply:
Did you get to the bottom of this? That's definitely a problem.

Reply:
Not entirely.. the difference is that the whitespace before the `/>` is not classified as a tag delimeter any more, which seems completely fine to me, but I can't directly explain why it changed now. The html textmate files in this test project were not up to date with our real ones though 🤷‍♂️

Reply:
Oh! OK, it's just the whitespace. That's totally fine. I thought it was the `/>`.